bindings/node/test/autoEncrypter.test.js

```diff
@@ -110,6 +110,92 @@ describe('AutoEncrypter', function () {
     sandbox.restore();
   });
 
+  describe('#constructor', function () {
+    // This case is driver 4.9.0 and higher.
+    context('when mongodb exports BSON', function () {
+      context('when a bson option is provided', function () {
+        const bson = Object.assign({}, BSON);
```

Comment:
We clone the original `BSON` module and use the clone as the `bson` option so it does not have object equality and then matches.